<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncluirCD</class>
 <widget class="QDialog" name="IncluirCD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="line_cd_barra_preco">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="titulo_CD">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>INCLUIR NOVO CD</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cd_nome">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cd_preco">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Preço :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cd_num_faixas">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>N° de Faixas:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_cd_barra_nome">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="botoes_cd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_cd_barra_num_faixas">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
